src/account/account.cairo

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-// OpenZeppelin Contracts for Cairo v0.8.0 (account/account.cairo)
+// OpenZeppelin Contracts for Cairo vX.Y.Z (account/account.cairo)
```

Comment:
I'm not sure we ever settled on this pattern. What was the advantage with it?

Reply:
We definitely did not settle it. The discussion is here: https://github.com/OpenZeppelin/cairo-contracts/pull/811#discussion_r1386568230

And I had the incorrect idea on adding the XYZ, so I'll switch it back